web/beacon-app/package.json

```diff
@@ -1,6 +1,11 @@
 {
   "name": "beacon-app",
   "version": "0.2.0",
+  "description": "User UI for Ensign.",
```

Comment:
This was in the setup for the ensign-user-ui, so this was added for consistency.